client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -527,6 +529,40 @@ export class PluginsList extends Component {
 		}
 	};
 
+	updateAllPluginsDialog = () => {
+		const { pluginUpdateCount, translate } = this.props;
+
+		const siteDetails = this.props.selectedSiteSlug ? this.props.selectedSiteSlug : 'all sites';
+
+		const translationArgs = {
+			args: { pluginUpdateCount },
+			count: pluginUpdateCount,
+		};
+
+		acceptDialog(
```

Comment:
Now that we have a new style for the confirmation popup, can we use those styles here?